### DISCUSSION TITLE
Show message on the remote when QR command received

### DISCUSSION THREAD

**Discussion by kouyoumin** [gopro/labs#613] [Feature Requests]
The camera shows an icon on the screen when QR command received.
But it's not easy to see the screen when camera is neck mounted.
It would be nice if something can be shown on the remote so I can confirm if QR scanned successfully, without taking off the camera.

**Comment by dnewman-gpsw** [comment]
That is not likely.  Labs hacks the camera, I expect the screens on the remote when requires changing its firmware.

**Reply by kouyoumin** [reply]
Would flashing its LED be easier?

**Reply by dnewman-gpsw** [reply]
No idea, I would have to learn how that would be done.
